src/main/java/christmas/constants/message/ErrorMessage.java

```diff
@@ -0,0 +1,19 @@
+package christmas.constants.message;
+
+public enum ErrorMessage {
+    ERROR("[ERROR] "),
+    INVALID_DATE(ERROR.getMessage() + "유효하지 않은 날짜입니다. 다시 입력해 주세요."),
+    INVALID_ORDER(ERROR.getMessage() + "유효하지 않은 주문입니다. 다시 입력해 주세요."),
+    MAXIMUM_ORDER(ERROR.getMessage() + "메뉴는 한 번에 최대 20개까지만 주문할 수 있습니다."),
+    ONLY_DRINKS(ERROR.getMessage() + "음료만 주문 시, 주문할 수 없습니다.");
+
+    private final String message;
+
+    ErrorMessage(String message) {
+        this.message = message;
+    }
```

Comment:
에러 메시지의 prefix를 enum들 사이에 위치 시키셨네요! 

다만 ErrorMessage들과 에러 메시지의 prefix는 분리시키는게 좋다고 생각합니다. 혼자만 성격이 다른 아이가 붕 뜨는 것 같다는 개인적인 생각이네요 🤔 

 ```suggestion
public enum ErrorMessage {
    INVALID_DATE(ERROR.getMessage() + "유효하지 않은 날짜입니다. 다시 입력해 주세요."),
    INVALID_ORDER(ERROR.getMessage() + "유효하지 않은 주문입니다. 다시 입력해 주세요."),
    MAXIMUM_ORDER(ERROR.getMessage() + "메뉴는 한 번에 최대 20개까지만 주문할 수 있습니다."),
    ONLY_DRINKS(ERROR.getMessage() + "음료만 주문 시, 주문할 수 없습니다.");
    private static final String ErrorMessagePrefix = "[ERROR] "
    private final String message;

    ErrorMessage(String message) {
        this.message = message;
    }
```

Reply:
오.. ERROR.getMessage() 대신에 private static final String ErrorMessagePrefix = "[ERROR] "로 정의한 ErrorMessagePrefix를 사용해보는 게  말씀이시죠?

public enum ErrorMessage {
   INVALID_DATE(ErrorMessagePrefix + "유효하지 않은 날짜입니다. 다시 입력해 주세요.")
   
이런 식으로요? 더 깔끔해질 것 같아요. 다음엔 적용해보겠습니다!

Reply:
저 같은 경우는 포맷 매칭 방법을 사용했어요.
뭔가 나중에 에러문을 추가할 때, 에러 prefix를 일일이 추가해야할 것 같더라구요.
그래서 에러문만 작성한 뒤에
getMessage 함수에서 "[ERROR] %s"를 포맷 메시지로 매칭 시켜줬어요.

Reply:
좋은 방법 같아요! 감사합니다👍